signale.js

```diff
@@ -257,6 +257,19 @@ class Signale {
     this._log(this._buildSignale(this._types[type], ...messageObj), this._types[type].stream);
   }
 
+  _padSpace(str, targetLength) {
+    if (String.prototype.padEnd) {
+      return str.padEnd(targetLength);
+    }
+    targetLength >>= 0; // Floor if number or convert non-number to 0;
```

Comment:
Bitwise operators are really handy, but in order to increase the transparency of what is taking place, it could be replaced with the following;

```js
targetLength = parseInt(targetLength, 10) || 0;
```

also, at the same time, we can cast the given `str` to String and move both arguments to the top of the function's scope;

```js
_padSpace(str, targetLength) {
  str = String(str);
  targetLength = parseInt(targetLength, 10) || 0;
  //...
}
```

Reply:
@klauscfhq But if you think the performance `bitwise` is faster than regular one https://jsperf.com/rounding-numbers-down/3